BUILD

```diff
@@ -2001,6 +2001,8 @@ cc_library(
     "src/compiler/node_generator_helpers.h",
     "src/compiler/objective_c_generator.h",
     "src/compiler/objective_c_generator_helpers.h",
+    "src/compiler/php_generator.h",
```

Comment:
nit: in case there's a lot of changes to the generated files, it is better to put all the generated changes in a separate commit, because that makes reviewing easier.